src/Widgets/CanvasItem.vala

```diff
@@ -136,4 +151,12 @@ public class GtkCanvas.CanvasItem : Clutter.Actor {
 
         updated ();
     }
+
+    internal void apply_rotation (double rotation) {
```

Comment:
This was made into a property :) so it can be deleted

https://github.com/Philip-Scott/libgtkcanvas/pull/13/files#diff-5f880ccfd30c8b8d210c6ab7dc02f8c3R74